demos/icon-toggle.html

```diff
@@ -24,9 +24,13 @@
         background: #303030;
       }
 
-      .mdc-theme--dark:not(.demo-color-combo) {
-        padding: 1rem;
-        padding-top: 0;
+      section.mdc-theme--dark:not(.demo-color-combo) {
+        padding-bottom: 1rem;
+      }
+
+      .mdc-icon-toggle {
```

Comment:
I'd maybe just put a `demo-wrapper` class around the icon toggles that give it `1rem` margins rather than mutate the icon toggle itself.